_data/docs.yml

```
@@ -144,6 +144,19 @@ apis:
       legacy: 0
       stable: 1
       nightly: 1
+  ucxx:
+    name: UCXX
+    path: ucxx
+    desc: 'UCXX is the Python interface for [UCX(<https://github.com/openucx/ucx), a low-level high-performance networking library. UCX and UCXX support several transport methods including InfiniBand and NVLink while still using traditional networking protocols like TCP.'
```

```suggestion
    desc: 'UCXX is the Python interface for [UCX](<https://github.com/openucx/ucx), a low-level high-performance networking library. UCX and UCXX support several transport methods including InfiniBand and NVLink while still using traditional networking protocols like TCP.'
```

Sorry, missed this in my previous reviews.  This link is not rendering correctly.